worker/handler.ts

```diff
@@ -88,12 +88,25 @@ export function extractAltLineNumberReference(
   };
 }
 
-function proxyFile(url: URL, remoteUrl: string, request: Request) {
+async function proxyFile(
+  url: URL,
+  remoteUrl: string,
+  request: Request,
+): Promise<Response> {
   const init = {
     method: request.method,
     headers: request.headers,
   };
   const urlR = remoteUrl + url.pathname;
   const modifiedRequest = new Request(urlR, init);
-  return fetch(modifiedRequest);
+  let lastErr;
+  for (let i = 0; i < 3; i++) {
+    try {
+      return await fetch(modifiedRequest);
+    } catch (err) {
+      console.warn("retrying on proxy error", err);
```

Comment:
maybe a little delay here?

Reply:
Delay should not be needed - it would only slow down the user connection. The failure case we are trying to fix by this is not timing based, so an extra deploy would not help.

Reply:
I think we should check at this point whether the error is what we expect.
But I guess it needs to land so we can run it. So add a todo?